modules/sff/module/inc/sff/8472.h

```diff
@@ -989,8 +992,10 @@ _sff8472_media_sfp28_cr(const uint8_t* idprom)
     /* module should be sfp */
     if (!SFF8472_MODULE_SFP(idprom)) return 0;
 
-    if ((idprom[3] & SFF8472_CC3_INF_1X_CU_PASSIVE) == 0) return 0;
```

Comment:
Do the new checks in idprom[36] cover the removed check in idprom[3]?

Reply:
@jnealtowns : Yes. I had a conversation with Vera Koleva who is the point of contact on SFF-8472 Rev 12.3 and as per his suggestion, idprom[36] is sufficient to drive of 25G CR transceivers. 
Please refer to Table 4.4 in SFF-8024 Rev 4.7. New Extended Specification Compliance codes have been introduced for 25G CR.

Reply:
Thanks @ashutosh-arrcus, sounds good to me.